tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -424,14 +424,10 @@ def _test_import_path(remote_command_executor, mount_dir):
     assert_that(result.stdout).is_equal_to("Downloaded by FSx Lustre")
 
 
-def _test_fsx_lustre_correctly_shared(scheduler_commands, remote_command_executor, mount_dir):
+def test_fsx_lustre_correctly_shared(scheduler_commands, remote_command_executor, mount_dir):
```

Comment:
Why did you remove the underscore prefix?
The underscore was a way to avoid considering this function as a "test" function, I think we should restore it because it's an utility function called by other real tests.

We can rename it to "assert_*" to avoid confusion.
The same is true for test_fsx_lustre_correctly_mounted.